react/features/conference/components/web/InviteMore.js

```diff
@@ -74,7 +81,8 @@ function mapStateToProps(state) {
     const hide = interfaceConfig.HIDE_INVITE_MORE_HEADER;
 
     return {
-        _visible: isToolboxVisible(state) && isButtonEnabled('invite', state) && isAlone && !hide
+        shouldShow: isButtonEnabled('invite', state) && isAlone && !hide,
+        toolboxVisible: isToolboxVisible(state)
```

Comment:
Is there a case when this is false and the above is true? IIRC we made it so the toolbar is always displayed when you are alone.

Reply:
From my testing, the toolbar still has the docking behaviour even when alone, but in this situation, I need to know whether the toolbar is displayed or not, separately, in order to add a class in the `render()` method.